Dockerfile

```
@@ -0,0 +1,31 @@
+FROM ubuntu:latest
+
+ENV TZ=Europe/Kiev
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    cmake \
+    g++ \
+    build-essential \
+    gperf \
+    openssl \
+    libssl-dev \
+    zlib1g-dev \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+
+RUN git clone --recursive https://github.com/tdlib/telegram-bot-api.git && \
+    cd telegram-bot-api/ && \
+    mkdir build && \
+    cd build
+
+WORKDIR telegram-bot-api/build
+
+RUN cmake .. -DCMAKE_BUILD_TYPE=Release \
+    && cmake --build . --target install
```

Shouldn't we use more than one CPU core to build the project? I think the easiest way to achieve parallel compilation, but keep the number of cores flexible, is to update this as:
```suggestion
RUN cmake .. -DCMAKE_BUILD_TYPE=Release \
    && cmake --build . --target install -- -j`nproc`
```
(which is going to make the compiler use all the cores by default) and control the number of cores available to docker (for example `docker build ... --cpuset-cpus 0-3` will let using first 4 cores)